app/models/sponsor.rb

```diff
@@ -2,6 +2,8 @@ class Sponsor < ActiveRecord::Base
   include Initializer
   NEW_CITY_MENU_OPTION = '**Add New**'
   PAYMENT_PLANS = ['Monthly', 'Every Two Months', 'Every Four Months', 'Every Six Months', 'Annually', 'Other']
+  PRIORITY_COUNTRIES= %w(SA TR AE GB)
+  EXCLUDED_COUNTRYS= %w(IL BQ BV IO KY CX CC CW FK TF GG HM IM JE YT AN NF NO PN BL SH MF PM SX GS SS SJ TK TC UM AX)
```

Comment:
To clarify (I misread your comment) the option I prefer is
"change `en_ar_country` to return only the English version on a missing translation."